### PR TITLE
update NodeUpgradeJob Image description

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
update NodeUpgradeJob Image description: if user specify the image tag, kubeedge will overwrite it with the spec.Version
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/nodeupgradejobcontroller/controller/downstream.go [hunk 1]
```diff
@@ -147,6 +147,22 @@ func (dc *DownstreamController) nodeUpgradeJobAdded(upgrade *v1alpha1.NodeUpgrad
...
+		if err != nil {
+			klog.Errorf("Image format is not right: %v", err)
+			return
+		}
+	} else {
+		repo = "kubeedge/installation-package"
+	}
+	imageTag := upgrade.Spec.Version
+	image := fmt.Sprintf("%s:%s", repo, imageTag)
+
```
Comment (wackxu) [inline cloud/pkg/nodeupgradejobcontroller/controller/downstream.go:165]:
```suggestion
	var repo string
	var err error
	repo := "kubeedge/installation-package"
	if upgrade.Spec.Image != "" {
		repo, err = GetImageRepo(upgrade.Spec.Image)
		if err != nil {
			klog.Errorf("Image format is not right: %v", err)
			return
		}
	}
	imageTag := upgrade.Spec.Version
	image := fmt.Sprintf("%s:%s", repo, imageTag)
```

Comment (gy95) [reply cloud/pkg/nodeupgradejobcontroller/controller/downstream.go:164]: fixed

**Comment by gy95** [comment]
cc @fisherxu  PTAL

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**